src/lib/abacus/dydxChainTransactions.ts

```diff
@@ -267,30 +277,24 @@ class DydxChainTransactions implements AbacusDYDXChainTransactionsProtocol {
       throw new Error('Missing compositeClient or localWallet');
     }
 
-    const {
-      orderId,
-      subaccountNumber,
-      clientId,
-      orderFlags,
-      clobPairId,
-      goodTilBlock,
-      goodTilBlockTime,
-    } = params ?? {};
+    const { subaccountNumber, clientId, orderFlags, clobPairId, goodTilBlock, goodTilBlockTime } =
+      params ?? {};
 
     try {
       const tx = await this.compositeClient?.cancelRawOrder(
         new SubaccountClient(this.localWallet, subaccountNumber),
         clientId,
         orderFlags,
         clobPairId,
-        goodTilBlock || undefined,
-        goodTilBlockTime || undefined
+        goodTilBlock ?? undefined,
+        goodTilBlockTime ?? undefined
```

Comment:
I think these were purposely left as `||` for the falsiness. It is to handle `0`

Reply:
Line 240 ish has another goodTilBlock ?? undefined from a a while ago. Should that be changed too?